Client/src/Components/DashboardMenu/index.jsx

```diff
@@ -0,0 +1,57 @@
+import language from '@mui/icons-material/language';
+import report_gmailerrorred from '@mui/icons-material/reportGmailerrorred';
+import sensors from '@mui/icons-material/sensors';
+import settings from '@mui/icons-material/settings';
+import all_inclusive from '@mui/icons-material/AllInclusive';
```

Comment:
These import statements don't work on my end.

I had a look in the documentation, and for the sensors icon for example it looks like it should be:

`import SensorsIcon from '@mui/icons-material/Sensors';`

Can you double check these?